kmq.js

```diff
@@ -70,6 +73,47 @@ client.on("message", (message) => {
     }
 });
 
+// Usage: `!help [action]` or `!help`
+const help = (message, action) => {
+    let embed_title = "";
```

Comment:
Going forward, we should use camelCase for variables but I'll just fix this manually